### DISCUSSION TITLE
Do not automatically switch game modes when jumping to an osu!standard beatmap

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#21404] [Ideas] [answered]
When playing in an alternate game mode, it can be pretty jarring if the game switches back to standard, even though the beatmap is playable in the current game mode. I would suggest setting an exception to auto-switching in this case.

https://user-images.githubusercontent.com/15932372/203771961-254f3a26-cc0f-4100-9372-672a96fa2a98.mp4

**Comment by Doksha** [comment, marked as answer]
probably should edit title to include the fact that you are using converted beatmaps; (ie. "Do not auto switch gamemodes to standard if "Show converted" is enabled")

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/issues/21415
